test/index.test.js

```diff
@@ -29,7 +29,7 @@ describe('mongodb-connection-model', function() {
       it('should return the correct URL for the driver', function() {
         var c = new Connection();
         assert.equal(c.driver_url,
-          'mongodb://localhost:27017/?slaveOk=true&ssl=prefer');
+          'mongodb://localhost:27017/?slaveOk=true');
```

Comment:
not sure about what `ssl=prefer` does here and why we want to remove it. I'll refer to @imlucas.